editor/src/clj/editor/material.clj

```diff
@@ -439,21 +459,21 @@ there is no way a user can know what the generated id will be for older shaders.
 
 (defn- filter-mode-min->gl [filter-min default-tex-params]
   (case filter-min
-    :filter-mode-min-default (:min-filter default-tex-params)
+    :filter-mode-min-default (or (:min-filter default-tex-params) GL2/GL_NEAREST_MIPMAP_LINEAR)
     :filter-mode-min-nearest GL2/GL_NEAREST
     :filter-mode-min-linear GL2/GL_LINEAR
     :filter-mode-min-nearest-mipmap-nearest GL2/GL_NEAREST_MIPMAP_NEAREST
     :filter-mode-min-nearest-mipmap-linear GL2/GL_NEAREST_MIPMAP_LINEAR
     :filter-mode-min-linear-mipmap-nearest GL2/GL_LINEAR_MIPMAP_NEAREST
     :filter-mode-min-linear-mipmap-linear GL2/GL_LINEAR_MIPMAP_LINEAR
-    nil))
+    GL2/GL_NEAREST_MIPMAP_LINEAR))
 
 (defn- filter-mode-mag->gl [filter-mag default-tex-params]
   (case filter-mag
-    :filter-mode-mag-default (:mag-filter default-tex-params)
+    :filter-mode-mag-default (or (:mag-filter default-tex-params) GL2/GL_LINEAR)
     :filter-mode-mag-nearest GL2/GL_NEAREST
     :filter-mode-mag-linear GL2/GL_LINEAR
-    nil))
+    GL2/GL_LINEAR))
```

Comment:
Always produce a valid default from these. `GL_LINEAR` is the default filtering mode in OpenGL.